app.js

```diff
@@ -41,11 +58,32 @@ app.use(cookieParser);
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
+app.use(AUTH_URL, authRouter);
+
+
+// Initialize Passport and restore authentication state, if any, from the
+// session.
+app.use(passport.initialize());
+app.use(passport.session());
+app.use(LOGIN_URL,
+  passport.authenticate('local', { failureRedirect: AUTH_URL }),
+  loginRouter
+);
+
+app.use(checkToken);
 app.use(USERS_URL, userRouter);
 app.use(PRODUCTS_URL, productRouter);
 
 app.get('/', (req, res) => {
-  res.send(`Available endpoints are: ${PRODUCTS_URL}, ${PRODUCT_URL}, ${REVIEWS_URL}, ${USERS_URL}`);
+  res.send(`Available endpoints are: ${PRODUCTS_URL}, ${PRODUCT_URL}, ${REVIEWS_URL}, ${USERS_URL}, ${AUTH_URL}, ${LOGIN_URL}`);
```

Comment:
you can kind of automate this part for yourself.

Put it in another "router" or something and import everything likewise:

```javascript
import * as endpoints from './endpoints';

...
res.send(`Available endpoints are: ${endpoints}`);
...
```
or just map over the array :)

Reply:
Good point. Will fix in the next PR.